app/controllers/proposal/collection_draft_proposals_controller.rb

```diff
@@ -226,6 +230,54 @@ def submitter_from_resource
       { name: "#{proposal_urs_user['first_name']} #{proposal_urs_user['last_name']}", email: proposal_urs_user['email_address'] } unless proposal_urs_user.blank?
     end
 
+    # Get urs users and get emails from urs for approvers
+    # Returns array of hashes for each user
+    def get_approver_emails
+      provider_targets = if Rails.env.production?
```

Comment:
Wrote MMT-2130 to move these to <environment>.rb files (needs to be done in other locations as well).